doc/whats_new.rst

```diff
@@ -38,6 +40,12 @@ API
 
     - Make the goodness of fit (GOF) of the dipoles returned by :func:`mne.beamformer.rap_music` consistent with the GOF of dipoles returned by :func:`mne.fit_dipole` by `Alex Gramfort`_.
 
+    - :class:`mne.decoding.SlidingEstimator` will now replace `mne.decoding.TimeDecoding` to make it generic and fully compatible with scikit-learn, by `Jean-Remi King`_ and `Alexandre Gramfort`_
+
+    - :class:`mne.decoding.GeneralizingEstimator` will now replace `mne.decoding.GeneralizationAcrossTime` to make it generic and fully compatible with scikit-learn, by `Jean-Remi King`_ and `Alexandre Gramfort`_
+
+    - :func:`mne.viz.decoding.plot_gat_times`, :func:`mne.viz.decoding.plot_gat_matrix` are now deprecated. Use matplotlib instead as shown in the examples, by `Jean-Remi King`_ and `Alexandre Gramfort`_
```

Comment:
if they're deprecated don't link to them, they should be removed from doc (but they haven't been?) and so the link will fail